app/components/section-work.hbs

```diff
@@ -26,6 +27,10 @@
                 <label for="work-use-ember-no">
                   No
                 </label>
+                <input type="radio" id="work-use-ember-idk" name="work-use-ember" />
```

Comment:
I removed this answer last night because of the wording of the question, but if we re-work the question for clarity, then this option might make sense. What do you think?

Reply:
I think it's definitely appropriate now and normal to not know whether or not your workplace uses Ember, especially at larger companies. For the related questioning, I'm thinking:
- Do you personally work on Ember at work? Yes/No (no IDK)
YES ->
- How often do you work on Ember at work [All the time, Most of the Time ... Rarely] (no never or IDK)

Reply:
I've taken out the Yes / No leader to keep it shorter but added the clarity in the question and 'Never' option there. 'Never / I do not personally use Ember at work'